### PR TITLE
bpmn-moddle Type Definitions

### PR DESCRIPTION
**Pull request by haydos404**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/bpmn-moddle/tslint.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
```
Comment (paulvanbrenk) [inline types/bpmn-moddle/tslint.json]: For a new definition you shouldn't need to disable all these rules.

**Review by paulvanbrenk — CHANGES_REQUESTED** [review]
Don't disable all the style rules, if there's problematic lines disable rules on a per line basis.

File: types/bpmn-moddle/tslint.json (hunk 1 above)
Comment (haydos89) [reply types/bpmn-moddle/tslint.json]: No worries. I removed all the special rules and specifically only extended dt.json in the latest commit.

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR has unaddressed comments and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**